README.md

```diff
@@ -127,6 +128,22 @@ For example, if you would like to cache a user, you might want to cache them by
 
 If no cacheKey is passed in, one will be generated by serializing and hashing the method name, arguments, and context in which the method was called. This will not allow you to reliably clear caches, but is available as a convenience.
 
+##### CacheStrategy
+
+A custom implementation of `CacheStrategy` can be passed in to replace the default strategy. The default strategy will always return the cached value, or call the method and cache the result otherwise. If you want to let's say update the cache before it's time to live end's you can make your own implementation of `CacheStrategy` like this:
```

Comment:
```suggestion
A custom implementation of `CacheStrategy` can be passed in to replace the default strategy. The default strategy will always return the cached value, or call the method and cache the result otherwise. If you want to update the cache before its time to live ends, you can implement your own `CacheStrategy` like this:
```